Orange/widgets/visualize/tests/test_owsieve.py

```diff
@@ -42,13 +86,12 @@ def test_missing_values(self):
         data = Table(Domain(attrs, class_var), X, np.array([np.nan] * 6))
         self.send_signal(self.widget.Inputs.data, data)
 
-    def test_keyerror(self):
+    def test_single_line(self):
```

Comment:
+1 for changing the name. You can also remove these GH-xxxx tags. They notify what was fixed when a function or a test was written, but this belongs to commit messages, not here. In particular in the code of the widget, in this case in method `sparse_to_dense`; the fact that the function was written to fix GH-2260 is not relevant for the future.